ParkingZoneApp/Areas/Admin/Views/ParkingSlot/Index.cshtml

```diff
@@ -0,0 +1,54 @@
+﻿@addTagHelper *, Microsoft.AspNetCore.Mvc.TagHelpers
+@model IEnumerable<ParkingZoneApp.ViewModels.ParkingSlotViewModels.ListOfParkingSlotsVM>
+
+@{
+    ViewData["Title"] = "Index";
+    Layout = "~/Views/Shared/_Layout.cshtml";
+}
+
+<h1>Index</h1>
+
+<p>
+    <a asp-controller="ParkingSlot" asp-action="Create" class="btn btn-primary">Create New</a>
+</p>
+<table class="table">
+    <thead>
+        <tr>
+            <th>
+                @Html.DisplayNameFor(model => model.Number)
+            </th>
+            <th>
+                @Html.DisplayNameFor(model => model.IsAvailableForBooking)
+            </th>
+            <th>
+                @Html.DisplayNameFor(model => model.Category)
+            </th>  
+            <th>
+                Actions
```

Comment:
Why did you add it?

Reply:
Actually Abdulloh bro was ok with this, coz he also added this